kg_microbe/transform_utils/constants.py

```diff
@@ -190,20 +190,20 @@
 MEDIUM_TO_INGREDIENT_EDGE = "biolink:has_part"  # Could also be has_constituent/has_participant
 MEDIUM_TO_SOLUTION_EDGE = "biolink:has_part"
 NCBI_TO_SHAPE_EDGE = "biolink:has_phenotype"  # [org_name -> cell_shape, metabolism]
-NCBI_TO_CARBON_SUBSTRATE_EDGE = "METPO:2000006"  # "uses as carbon source" [org_name -> carbon_substrate]
+NCBI_TO_CARBON_SUBSTRATE_EDGE = (
+    "METPO:2000006"  # "uses as carbon source" [org_name -> carbon_substrate]
+)
 NCBI_TO_ISOLATION_SOURCE_EDGE = "biolink:location_of"  # [org -> isolation_source]
 NCBI_TO_METABOLISM_EDGE = "biolink:has_phenotype"  # [org -> metabolism]
 NCBI_TO_PATHWAY_EDGE = "biolink:capable_of"  # # [org -> pathway]
 CHEBI_TO_ROLE_EDGE = "biolink:has_chemical_role"
-NCBI_TO_METABOLITE_UTILIZATION_EDGE = "biolink:consumes"  # [org -> metabolite_utilization]
 NCBI_TO_ENZYME_EDGE = "biolink:capable_of"  # [org -> enzyme]
 ASSAY_TO_NCBI_EDGE = "biolink:assesses"  # [org -> assay]
 MEDIUM_TO_METABOLITE_EDGE = "biolink:assesses"  # [org -> assay]
 NCBI_TO_ASSAY_EDGE = "biolink:is_assessed_by"  # [org -> assay]
-NCBI_TO_METABOLITE_PRODUCTION_EDGE = "biolink:produces"
 ENZYME_TO_ASSAY_EDGE = "biolink:is_assessed_by"  # [enzyme -> assay]
 SUBSTRATE_TO_ASSAY_EDGE = "biolink:occurs_in"  # [substrate -> assay]
-ENZYME_TO_SUBSTRATE_EDGE = "biolink:consumes"  # [substrate -> enzyme]
+ENZYME_TO_SUBSTRATE_EDGE = "biolink:has_input"  # [enzyme -> substrate]
```

Comment:
Is this the best choice of biolink predicates for this relation?

Documentation page for `biolink:has_input`: https://biolink.github.io/biolink-model/has_input/ says that this predicate should be used for when the domain is a "process" and the range is an "entity". Here we are using it to represent the relation between two entities (enzyme and substrate).